yaml/lev.yaml

```diff
@@ -1,20 +1,23 @@
 person:
    - name: lev
-     age: 18
-     isPerson: false
+   - age: 18
+   - isPerson: false
   friends:
```

Comment:
person is an array,
friends is not a part of person if not marked with an "-". then it is bad spacing.